### DISCUSSION TITLE
Support to create non PHP sites

### DISCUSSION THREAD

**Discussion by maxcelos** [vitodeploy/vito#205] [Ideas]
Besides my Laravel app, I need to host a static documentation (html) and a webrtc server (nodeJs). I did manage to solve by installing pm2 and updating the vhost file, but would be very nice to be able to install it from dashboard. 🙂 

Also, it gets the PHP logo in the sites list, being a nodejs app. 😅

**Comment by saeedvaziry** [comment]
@maxcelos I already have plan for this but not in the close future.

There is already a PR opened (#190) but needs to be worked on.
